projection/account/account.go

```diff
@@ -109,72 +206,102 @@ func (projection *Account) HandleEvents(height int64, events []event_entity.Even
 	return nil
 }
 
-func (projection *Account) handleAccountCreatedEvent(accountsView account_view.Accounts, event *event_usecase.AccountTransferred) error {
+func (projection *Account) handleGenesisCreatedEvent(accountsView account_view.Accounts, event *event_usecase.GenesisCreated) error {
 
-	recipienterr := projection.writeAccountInfo(accountsView, event.Recipient)
-	if recipienterr != nil {
-		return recipienterr
-	}
+	for _, account := range event.Genesis.AppState.Bank.Balances {
+		for _, accountBalance := range account.Coins {
```

Comment:
Do we need to cater for vesting balance here?

Reply:
For example using Crypto.org Chain mainnet genesis, you can see some of the account has `DelayedVesting` and the balance will appear at `Gensis.AppState.Bank.Balances`

https://mainnet.crypto.org:26657/genesis

Reply:
i think `Genesis.AppState.Bank.Balances` would be enough since catering `DelayedVesting` would result in double counted

Reply:
I think for the `DelayedVesting` account, when it reached the vesting date, there will be `coin*` event for that.

Reply:
If those Coin is under `DelayedVesting`, should we add them as `UsableBalance` here?

Reply:
Right, let me put some effort on it.